demos/meltdown_ss.cc

```diff
@@ -97,9 +102,11 @@ static char leak_byte(size_t offset) {
     // segment that points to private data.
 
     // PL = 3, local_table = 1 * 4, index = 0 * 8.
-    size_t fs_backup = ExchangeFS(3 + 4);
+    int fs_backup = 3 + 4;
+    ExchangeFS(&fs_backup);
```

Comment:
The previous API (take new, return old) was a bit easier to read and understand. Can we go back to that?

Reply:
Done.